plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/steps/embedded-kyc.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { LoadError } from '@stripe/connect-js';
+
+/**
+ * Internal dependencies
+ */
+import { useOnboardingContext } from '../../../data/onboarding-context';
+import StripeSpinner from '../../../components/stripe-spinner';
+import BannerNotice from '../../../components/banner-notice';
+import { useBusinessVerificationContext } from '../data/business-verification-context';
+import { finalizeOnboarding, isPoEligible } from '../utils';
+import { EmbeddedAccountOnboarding } from '../components/embedded';
+
+interface Props {
+	continueKyc?: boolean;
+	collectPayoutRequirements?: boolean;
+}
+
+const EmbeddedKyc: React.FC< Props > = ( {
+	continueKyc = false,
+	collectPayoutRequirements = false,
+} ) => {
+	const { data } = useBusinessVerificationContext();
+	const { currentStep, navigateToNextStep } = useOnboardingContext();
+	const [ finalizingAccount, setFinalizingAccount ] = useState( false );
+	const [ isEligible, setIsEligible ] = useState< boolean | null >( null );
+	const [ loading, setLoading ] = useState( true );
+	const [ loadError, setLoadError ] = useState< LoadError | null >( null );
+	const fallbackUrl = currentStep?.actions?.kyc_fallback?.href ?? '';
+
+	// Fetch whether the account is eligible for progressive onboarding
+	useEffect( () => {
+		const checkEligibility = async () => {
+			const eligibility = await isPoEligible( data );
+			setIsEligible( eligibility );
+		};
+
+		if ( ! continueKyc ) {
+			checkEligibility();
+		} else {
+			setIsEligible( false );
+		}
+	}, [ continueKyc, data ] );
+
+	const handleStepChange = () => {
+		// To-Do: Track step change.
```

Comment:
Is this a correct To-Do that we need to complete? 🤔

Reply:
Yes, I have create an issue to handle the tracking implementation. https://github.com/woocommerce/woocommerce/issues/56826